packages/swap/src/routes/quote.ts

```diff
@@ -57,10 +54,8 @@ const quote = (io: Server) => {
       }
 
       const includedFees: SwapFee[] = [];
-      const ingressFee = await estimateIngressEgressFeeAssetAmount(
-        await getNativeIngressFee(query.srcAsset),
-        query.srcAsset.asset,
-      );
+      const ingressFee = await getNativeIngressFee(query.srcAsset);
```

Comment:
`native` is not correct anymore here 🙈 think we can rename them to `getIngressFee` and `getEgressFee`